pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/sql.tsx

```diff
@@ -49,6 +49,16 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph
+      title="SQL Query Errors"
+      sources={nodeSources}
+      tooltip={"The number of statements which returned a planning or runtime error."}
+    >
+      <Axis label="queries">
+        <Metric name="cr.node.sql.failure.count" title="Failures" nonNegativeRate />
```

Comment:
I think it may be confusing to use different terms for the chart title and the series title.  Maybe just s/Failures/Errors/?

Reply:
Done.